README.md

```diff
@@ -65,9 +66,24 @@ For an out-of-the-box setup, you can use my docker-compose recipe or a docker im
 
 For automating the analysis of your Scala projects, check out my sbt plugin [mwz/sbt-sonar](https://github.com/mwz/sbt-sonar).
 
+# Sonar-scanner properties
+The plugin exposes the following properties which can be passed to sonar-scanner when running an analysis:
+- *sonar.sources* (optional) - Scala source directory relative to the root of your project (defaults to `src/main/scala`)
```

Comment:
I'm almost 100% sure, that the `sonar.sources` property is mandatory.

Reply:
It looks to me like it falls back to `src/main/scala` if not set, see [here](https://github.com/mwz/sonar-scala/blob/3973e6a8b3857c06de7b6d996702eeb3e543e5e0/src/main/scala/com/mwz/sonar/scala/ScalaPlugin.scala#L54).

Reply:
Uhm, yeah that default value was a mistake of mine, you can't run a **sonar-scanner** in any project without that property, it is mandatory by sonar, as can be see [here](https://docs.sonarqube.org/display/SONAR/Analysis+Parameters#AnalysisParameters-ProjectConfiguration).

Also, just to confirm that, here is the output of running **sonar-scanner** in one of my projects after removing the `sonar.sources` _property_.

```bash
$ sonar-scanner
INFO: Scanner configuration file: /opt/sonar-scanner/conf/sonar-scanner.properties
INFO: Project root configuration file: /home/balmungsan/project/sonar-project.properties
INFO: SonarQube Scanner 3.0.3.778
INFO: Java 1.8.0_121 Oracle Corporation (64-bit)
INFO: Linux 4.4.0-43-Microsoft amd64
INFO: User cache: /home/balmungsan/.sonar/cache
INFO: Publish mode
INFO: Load global settings
INFO: Load global settings (done) | time=484ms
INFO: Server id: -----
INFO: User cache: /home/balmungsan/.sonar/cache
INFO: Load plugins index
INFO: Load plugins index (done) | time=264ms
INFO: SonarQube server 6.7.2
INFO: Default locale: "en_US", source code encoding: "UTF-8"
INFO: Process project properties
INFO: ------------------------------------------------------------------------
INFO: EXECUTION FAILURE
INFO: ------------------------------------------------------------------------
INFO: Total time: 8.382s
INFO: Final Memory: 44M/279M
INFO: ------------------------------------------------------------------------
ERROR: Error during SonarQube Scanner execution
ERROR: You must define the following mandatory properties for 'anthill-agent': sonar.sources
ERROR:
ERROR: Re-run SonarQube Scanner using the -X switch to enable full debug logging.
```

So it really doesn't matter if the sensor defines a default value for it, the reason that default value exist, its that I first tried to use the _property key_ `sonar.scala.sources`, but then sonar complained because `sonar.sources` was not defines (as showed above), So I decided to use that instead former, because what was the propose of having two different properties that will always had the same value. But then when I revert the changes I forgot to remove the default value.

Note that this raise a minor problem, that line of code is misleading and should be changed, but changed for what ?, a direct access to the property as we know it should be always defined ? (maybe with a suppress warning annotation ?).

Reply:
Great, thanks for explaining this to me - I've removed the "optional" part of the `sonar.sources` property.

Reply:
:+1:, also thanks for the clarification in the **Credits** section, even when it was just a bad interpretation of mine.